README.md

```diff
@@ -2,40 +2,28 @@
 :closed_book: This is a book you need to buy yourself<br/>
 :book: This is a book free to download / online handbook<br/>
 :earth_africa: This is a link to an online article / blog post<br/>
-:books: This is a link to an online library of tecniques<br/>
+:books: This is a link to an online library of techniques<br/>
 
```

Comment:
Can we add a link to the "polish" readme somewhere in this "main" readme? We can use flag icon and description like "Resources in Polish" or something.

Reply:
I like the idea, on it :)